lldb/test/API/functionalities/param_entry_vals/basic_entry_values/main.cpp

```diff
@@ -52,9 +52,10 @@ __attribute__((noinline)) void func4_amb(int &sink, int x) {
   //%     expect_cmd_failure=True)
   //% self.filecheck("expr sink", "main.cpp","-check-prefix=FUNC4-EXPR",
   //%     expect_cmd_failure=True)
-  // FUNC4-EXPR-FAIL: couldn't get the value of variable x: Could not evaluate
-  // DW_OP_entry_value. FUNC4-EXPR: couldn't get the value of variable sink:
-  // Could not evaluate DW_OP_entry_value.
+  // FUNC4-EXPR-FAIL: couldn't get the value of variable x: could not evaluate
+  // DW_OP_entry_value: no matching call site param found FUNC4-EXPR: couldn't
+  // get the value of variable sink: could not evaluate DW_OP_entry_value: no
```

Comment:
There's a missing newline that disables part of the check.

Reply:
Ugh, this is clang-format wrapping the line. I'll need to wrap that between `clang-format off`.